daisy_workflows/image_import/suse/translate.py

```diff
@@ -91,7 +92,7 @@ def __init__(self, flavor, major, minor, products=None):
     _SuseRelease(
         flavor='sles',
         major='12',
-        minor='5',
+        minor='4|5',
```

Comment:
Shouldn't this be a separate _SuseRelease object?

Reply:
In terms of translation for BYOL, SLES 12.4, SLES for SAP 12.4, and SLES 12.5 are all the same (need the same product enabled and need the same packages). The difference is the license.